website/project/decorators.py

```diff
@@ -86,6 +86,12 @@ def wrapped(*args, **kwargs):
 
             _inject_nodes(kwargs)
 
+            if getattr(kwargs['node'], 'is_collection', True):
+                raise HTTPError(
+                    http.BAD_REQUEST,
```

Comment:
There's a similar error with retracted registrations that treated the same, so we were leaning towards matching that.